### ISSUE TITLE
Version should be 4.2.2 in package.json and missing v4.2.1 and v4.2.2 tags

### ISSUE THREAD

**Issue by tbouffard** [NaturalIntelligence/fast-xml-parser#567]
Note that the problem already occurred in the past: #539

Latest commit available in the master branch:

https://github.com/NaturalIntelligence/fast-xml-parser/blob/b4d30118c986bfc04ccc557ecccb326f67af02ea/package.json#L1-L6

Currently, the latest version available on [npmjs](https://www.npmjs.com/package/fast-xml-parser) is `4.2.2` but there is no related tag in the repository.
The tag for the 4.2.1 version is named `4.2.1`, whereas all other tags are prefixed by `v`.

![image](https://user-images.githubusercontent.com/27200110/234826225-9e915dc0-ccc7-4bbd-9121-9a1a7c8f377e.png)
_https://github.com/NaturalIntelligence/fast-xml-parser/tags_

This makes the full changelog of the version hard to follow.
The only way to get the details of the changes is to access to this url: https://github.com/NaturalIntelligence/fast-xml-parser/compare/4.2.1...master. But, the commits list will change in the future as far as new commits are pushed.

Here is the full changelog view:
![image](https://user-images.githubusercontent.com/27200110/234826771-099f41c2-71cc-447d-8d9b-663cd07dfb95.png)

**Would you like to work on this issue?**

- [ ] Yes
- [x] No

**Comment by amitguptagwl** [comment]
Thanks for highlighting. This is now available.

**Comment by tbouffard** [comment]
Thanks for the quick fix.
